### PR TITLE
WF-61451 - Add information about to edit content control text in the Word document

### PR DESCRIPTION
**Pull request by ManikandanRavichandranSync**
WF-61451 - Add information about to edit content control text in the Word document

### CODE REVIEW

**Comment by ManikandanRavichandranSync** [comment]
Rebuild

**Comment by DeepakRajSundar** [comment]
rebuild

**Comment by kannans24** [comment]
rebuild

File: File-Formats/DocIO/Working-With-Content-Controls.md [hunk 1]
```diff
@@ -3727,4 +3727,556 @@ document.Close();
 {% endhighlight %}
 {% endtabs %}
 
+## Edit Content Control Text
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-With-Content-Controls.md:3730]: Change the title as "Edit inline content text"

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-With-Content-Controls.md [hunk 2]
```diff
@@ -3727,4 +3727,556 @@ document.Close();
...
+    WCharacterFormat characterFormat = null;
+    foreach (ParagraphItem item in inlineContentControl.ParagraphItems)
+    {
+        if (item is WTextRange)
+        {
+            characterFormat = (item as WTextRange).CharacterFormat;
+            break;
+        }
+    }
+	//Remove exiting items and add new text range with required text
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-With-Content-Controls.md:3832]: Check the indent of xml comment

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**